### ISSUE TITLE
MircoDAGER Radio Programmer dont show right numbers

### ISSUE THREAD

**Issue by Sheppard85oc** [acemod/ACE3#5359]
**Arma 3 Version:** `1.77 stable
**CBA Version:** `3.4.0` (stable / dev + commit hash)
**ACE3 Version:** `3.10.1` (stable / dev + commit hash)

**Mods:**
```
- CBA_A3
- ace
Task Force Radio

**Description:**
The MircoDAGER Radio Programmer dont show numbers correlty 

**Steps to reproduce:**
If you have MircoDAGER Radio Programmer Equement it and press M do you see numbers not more

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer) / Virtual Arsenal

**Placed Modules:**
- None

**RPT log file:**
No RPT log but i can send a Pic of it

**Comment by jonpas** [comment]
Isn't that a TFAR item?

**Comment by Sheppard85oc** [comment]
I have it tested with mods CBA3 und just only TFRA but there no error make i it with CBA Task Force Radio and Ace its the Error there i make the Pic that you can see what i mean

![unbenannt](https://user-images.githubusercontent.com/26699856/28090947-6d622096-668e-11e7-89c8-81a6a8756c81.png)

**Comment by jonpas** [comment]
That's a GPS and has already been fixed for next release in #5337.

**Comment by Sheppard85oc** [comment]
I sorry for dupli report i have not find someone from this^^ Thanks
